app/views/catalogs/_logs.html.erb

```diff
@@ -75,3 +33,22 @@
     		$.SocketURL = "<%= Ind.websocket_tap %>";
      })($);
 	</script>
+
+<script src="https://cdn.socket.io/socket.io-1.2.0.js"></script>
+<script>
+  var socket = io.connect($.SocketURL);
+  console.log(socket);
+
+    $('.socket_disconnect').click(function(){
+         socket.disconnect();
+    });
+
+
+		socket.on("disconnect", function(){
+        console.log("client disconnected from server");
```

Comment:
Woops! Will refine and push it tomorrow.  :)
On 8 Oct 2015 22:14, "Kishorekumar Neelamegam" <notifications@github.com>
wrote:

> In app/views/catalogs/_logs.html.erb
> <https://github.com/megamsys/nilavu/pull/553#discussion_r41537570>:
>
> > @@ -75,3 +33,22 @@
> >      		$.SocketURL = "<%= Ind.websocket_tap %>";
> >       })($);
> >  	</script>
> > +
> > +<script src="https://cdn.socket.io/socket.io-1.2.0.js"></script>
> > +<script>
> > +  var socket = io.connect($.SocketURL);
> > +  console.log(socket);
> > +
> > +    $('.socket_disconnect').click(function(){
> > +         socket.disconnect();
> > +    });
> > +
> > +
> > +		socket.on("disconnect", function(){
> > +        console.log("client disconnected from server");
>
> Again console.log, please remove them.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/megamsys/nilavu/pull/553/files#r41537570>.
>